projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -186,10 +193,11 @@ export class KeyNavigationGridController implements OnDestroy {
     activeCell.setAttribute('tabindex', '0');
     this._activeCell = activeCell;
 
-    const items = getTabableItems(activeCell);
-    const item = activeCell.getAttribute('role') !== 'columnheader' && items[0] ? items[0] : activeCell;
+    const actionableItems = getTabableItems(activeCell);
+    const item =
+      activeCell.getAttribute('role') !== 'columnheader' && actionableItems[0] ? actionableItems[0] : activeCell;
```

Comment:
```suggestion
    const items = getTabableItems(activeCell);
    const item =
      activeCell.getAttribute('role') !== 'columnheader' && items[0] ? items[0] : activeCell;
```

I think this rename is not needed.

Reply:
I was personally confused by the name "items" while reading the code. Realised items are actionable-items inside the current cell, only while debugging.